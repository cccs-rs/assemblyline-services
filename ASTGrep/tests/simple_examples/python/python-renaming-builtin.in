var1 = (exec)
var2 = eval

var2("123+5")
var1("print('evil')")

Calculate, Negative, _floor, Run, _round, _frame, _math = exec, str, tuple, map, ord, globals, type

Calculate(aa)
Run("b")
_math("c")
Run(aaa, b)

def somefunc(var2 = var1):
    pass

O0O0o0oOOO0oOOoo0000o00O = binascii.unhexlify
O0O0o0oOOO0oOOoo0000o00O()

(var3, var4) = (exec, eval)

var3("123+5")
var4("123+5")
