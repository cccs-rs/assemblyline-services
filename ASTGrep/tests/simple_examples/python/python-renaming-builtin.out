var1 = (exec)
var2 = eval

123+5
print('evil')

Calculate, Negative, _floor, Run, _round, _frame, _math = exec, str, tuple, map, ord, globals, type

exec(aa)
map("b")
type("c")
map(aaa, b)

def somefunc(var2 = exec):
    pass

O0O0o0oOOO0oOOoo0000o00O = binascii.unhexlify
binascii.unhexlify()
